src/main/resources/db/migration/V1__Create_Beacons.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE IF NOT EXISTS beacons (
+    id SERIAL,
+    beaconType VARCHAR,
+    hexId VARCHAR,
+-- Would it be better to have hexId as Hex type for more efficient queries?
+-- Do we want to limit the length to 15 and 23?
```

Comment:
Would we want to limit length of hexIds? Guessing we'll be doing validation in the backend and/or frontend so would it be necessary?